wireapp/wire-android
# fix: Do not ask for external write permissions on android 13 and above

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

On devices running Android 13 (Tiramisu) and above, Google reworked the permissions requests logic so that it will not be needed to request the `WRITE_EXTERNAL_STORAGE` anymore. If this permission is still checked from that version onwards, it will always return `false`.

### Solutions
Remove the request for permissions to write in external storage as it is not needed when using a content resolver. That way we will unify the way to save files externally in both the gallery and the conversation view models. Also removed the permission from Manifest.

### Testing

#### How to Test

Have a device with Android 13 or above. Receive an asset. Try to save it and check it always succeeds

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
